### ISSUE TITLE
The formData issue

### ISSUE THREAD

**Issue by jamalkarami** [sneas/ionic-native-http-connection-backend#165]
### Subject of the issue
Hi everyone :)
I wich you are fine

I hava a problem with sending form data oject

I have installed the package as mentionned in the Readme file, and import NativeHttpModule and providers as following : 

```typescript
import { NativeHttpModule, NativeHttpBackend, NativeHttpFallback } from 'ionic-native-http-connection-backend';

@NgModule({
  declarations: [AppComponent],
  imports: [BrowserModule, IonicModule.forRoot(), AppRoutingModule, HttpClientModule,
    NgxPaginationModule,   
    NativeHttpModule
  ],
  providers: [
    { provide: RouteReuseStrategy, useClass: IonicRouteStrategy },
    {
      provide: HTTP_INTERCEPTORS,
      useClass: HttpInterceptorService,
      multi: true,
    },
    {
      provide : ErrorHandler, useClass : GlobalErrorHandler
    },
    {provide: HttpBackend, useClass: NativeHttpFallback, deps: [Platform, NativeHttpBackend, HttpXhrBackend]},
    Chooser  
  ],
  bootstrap: [AppComponent],
})
```

Get requests works fine, but when I do a post request uploading a formData object, it doesn't work, the request doesn't reach the back-end and doesn't return a error responce, juste timeout error

Knowing that all requests work fine with httpClient of angular

I have logged the request before launching it :

![image](https://user-images.githubusercontent.com/15126674/196930274-f936e755-0456-4930-a54c-fb3c745676b8.png)

Is there something missing in the request ?
what is the problem ? isn't formData supported yet in this package ?

I used this package to bypasse the CORS Policy,

Thanks for any solution or any other suggestions, 

And Thanks also for reading my issue.

### Environment
* version of angular : 14.0.0
* version of ionic-native-http-connection-backend : 10.0.0
* version of @ionic/angular : 6.1.9
* version of @awesome-cordova-plugins/http : 6.0.1

### Steps to reproduce
as Described above

### Expected behaviour
I expect a responce of saving successfully my form data in back-end

### Actual behaviour

doesn't reach the back-end and doesn't return a response

**Comment by sneas** [comment]
Hi @jamalkarami, that's unfortunate! This component supposes to accept `FormData` instances as a request body. Have you tried to debug troubleshoot the problem with this document https://github.com/sneas/ionic-native-http-connection-backend/blob/master/TROUBLESHOOTING.md ?

The request suppose to look like this:

```typescript
const formData = new FormData();
formData.set('a', 'b');

this.nativeHttp
  .sendRequest('https://httpbin.org/post', {
    method: 'post',
    data: formData,
    serializer: 'multipart',
  })
  .then(data => {
    console.log(
      'cordova-plugin-advanced-http is installed properly',
      data,
    );
  });
```

What do you see as a result?

**Comment by jamalkarami** [comment]
Hi @sneas, thanks for your reply,  yes it's unfortunate! 
I have tested to debug troubleshoot, This is the result : 

![image](https://user-images.githubusercontent.com/15126674/197153285-a69cafca-582c-4b60-887c-45ebd7f0d262.png)

I have used the same code request except two difference : 

**First** , I use formData.append instead of formData.set
Th second, I use formData to send not only simple data as your test, I use it to send both data and files

I have retried it again by matching your code : 
Here is the last code i tried that matches your code : 
`    const formData = new FormData();
    formData.set(`documentAdministratifDtos[0].typeDocument.id`, this.mDocumentAdministratif[0].typedocumentdto.id.toString())
    formData.set(`documentAdministratifDtos[0].modtifdto.id`, this.mDocumentAdministratif[0].modtifdto.id.toString())
    let dataResult = await fetch(this.mDocuments[0].dataURI);
    let blob = await dataResult.blob();

    formData.set('files', blob, this.mDocuments[0].name);
    this.nativeHttp
      .sendRequest('http://192.168.4.8:8081/apis/dossierAdministratif/user/ajouterNouvelleDemande', {
        method: 'post',
        data: formData,
        serializer: 'multipart',
        responseType : 'json'
      })
      .then(data => {
        console.log(
          'data saved successfuly',
          data,
        );
        loading.dismiss();
      }).catch(err => console.log("an error occured", err)
      );`
      
Since i don't declare the timeout in the request, I don't even get timeout response and no request reached in the bach-end,

Khnowing that I tested it with live reload in three réal devices of various version of android, respectively 7, 8, and 10 android versions. and getting the some issue.

I have read in the Readme file of the plugin Cordova Advanced Http in github the following : 

![image](https://user-images.githubusercontent.com/15126674/197161468-6b9eb985-9f20-4c0c-8596-b5f410642d31.png)

may the webview of this devices on which i have tested my app don't support formData.entreis as they said in this doc : 

https://github.com/silkimen/cordova-plugin-advanced-http/wiki/Web-APIs-required-for-Multipart-requests

Thanks @sneas for any other solution, suggestion or any informations
